doc/syscalls/README.md

```diff
@@ -105,11 +105,12 @@ stabilized or not (a "✓" indicates stability) in the Tock 1.0 release.
 
 ### Sensor ICs
 
-|1.0| Driver Number | Driver           | Description                                |
-|---|---------------|------------------|--------------------------------------------|
-|   | 0x70000       | TSL2561          | Light sensor                               |
-|   | 0x70001       | TMP006           | Temperature sensor                         |
-|   | 0x70004       | LPS25HB          | Pressure sensor                            |
+|1.0| Driver Number | Driver                    | Description                                |
+|---|---------------|---------------------------|--------------------------------------------|
+|   | 0x70000       | TSL2561                   | Light sensor                               |
+|   | 0x70001       | TMP006                    | Temperature sensor                         |
+|   | 0x70004       | LPS25HB                   | Pressure sensor                            |
+| ✓ | 0x70005       | [L3GD20](70005_l3gd20.md) | 3 axis gyroscope and temperature sensor    |
```

Comment:
```suggestion
|   | 0x70005       | [L3GD20](70005_l3gd20.md) | 3 axis gyroscope and temperature sensor    |
```

Reply:
It is in the readme, I removed the tick for 1.0.